specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/appconfiguration.json

```diff
@@ -939,6 +939,25 @@
         "encryption": {
           "$ref": "#/definitions/EncryptionProperties",
           "description": "The encryption settings of the configuration store."
+        },
+        "privateEndpointConnections": {
```

Comment:
Can we mark this as readonly to avoid breaking change?

Reply:
Yes, we can make this readonly. I'll wait for the reply on my other comment to make the change.

Reply:
Done.